internal/host/validator.go

```diff
@@ -957,6 +957,33 @@ func comparisonBuilder(value float64) string {
 	return equals
 }
 
+func (v *validator) generatePacketLossAdvisoryMessageForHost(c *validationContext, hostPacketLoss [] string) string {
+	message := "Actions:\n"
+	message += "1: Check if there are multiple devices on the same L2 network, if so then create a bond or disable all but one of the interfaces.\n"
+
+	var inventory models.Inventory
+	err := json.Unmarshal([]byte(c.host.Inventory), &inventory)
+	if err != nil {
+		v.log.WithError(err).Warnf("Could not parse inventory of host %s\n", *c.host.ID)
+	}
+	hostName := getRealHostname(c.host, &inventory)
+	if len(inventory.Interfaces) > 0 {
+		message = fmt.Sprintf("2: Please try the following commands on the host %s to investigate the packet loss further\n\n", hostName)
+		for _, intf := range inventory.Interfaces {
+			for _, hostPacketLossEntry := range hostPacketLoss {
+				entryParts := strings.Split(hostPacketLossEntry, " ")
```

Comment:
All this formatting and splitting is rather confusing, can we change `validatePacketLossForRole` to return structured data rather than a formatted string, so we can use it here rather than extracting it from an already formatted string?